bugsnag/django/__init__.py

```diff
@@ -26,14 +26,19 @@ def add_django_request_to_notification(notification):
             notification.context = "%s %s" % (request.method,
                                               request.path_info)
 
-    if hasattr(request, 'user') and request.user.is_authenticated():
-        try:
-            name = request.user.get_full_name()
-            email = getattr(request.user, 'email', None)
-            username = six.text_type(request.user.get_username())
-            notification.set_user(id=username, email=email, name=name)
-        except Exception:
-            bugsnag.logger.exception('Could not get user data')
+    if hasattr(request, 'user'):
+        if hasattr(request.user.is_authenticated, '__call__'):
```

Comment:
There is a `callable` method, maybe better to use that instead of checking for `__call__` method.

```python
if callable(request.user.is_authenticated):
```

Reply:
You're correct, I was following outdated information